### PR TITLE
tuya-panel-style-xxx

### PR DESCRIPTION
**Pull request by ZengBeauty**
## Description

Please provide a brief summary of the changes. If it is related to an issue, please bring the issue number, like `fixes #${number}`.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by youngjuning — APPROVED** [review]
LGTM~

File: example/tuya-components-styled/scripts/yalc.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# cd
+cd ../../packages/tuya-panel-kit
+
+for pkg in \
+tuya-panel-kit \
```
Comment (youngjuning) [inline example/tuya-components-styled/scripts/yalc.sh:5]: 可以维护到数组中

**Review by youngjuning — APPROVED** [review]
LGTM~

**Review by Saber2pr — APPROVED** [review]

File: example/tuya-components-styled/scripts/yalc.sh (hunk 1 above)
Comment (Saber2pr) [reply example/tuya-components-styled/scripts/yalc.sh:5]: 这块暂定，如果每个组件都需要发yalc，量起来后会很慢。先只考虑kit的yalc吧。

**Pull request merged**